### DISCUSSION TITLE
What version of webpack is supported to convert to vite?

### DISCUSSION THREAD

**Discussion by ahmad-reza619** [originjs/webpack-to-vite#89] [Q&A] [answered]
I have several old project using webpack, and i want to convert it to vite, but i don't know if my project can be converted or not. Is this project only support converting new webpack like v4 or v5 or is it able to convert up to webpack 2?

**Comment by Chieffo2021** [comment, marked as answer]
We encourage you to try out the tool to see if it works. In theory, our tool can convert any webpack project.

The `readme` docs list the conversion steps that are automatically performed by the tool and that need to be manually performed. If the conversion fails, you can manually perform the conversion according to the docs.

Anyway, there's no restriction on supporting a specific range of webpack versions, because the tool just imports your webpack configuration file and converts it.

Thanks to the fact that Vite integrates most of the features out of the box, we can discard most of the webpack configuration and convert only a few of the necessary configurations.
